ofi.sh

```diff
@@ -0,0 +1,38 @@
+package: ofi
+version: "%(tag_basename)s"
+tag: v1.7.0
+source: https://github.com/ofiwg/libfabric
+requires:
```

Comment:
There are no logs because aliBuild fails on this line, I guess `requires` can't be empty.

Reply:
removed it